DesktopModules/Vanjaro/Core/Library/Factories/PageFactory.cs

```diff
@@ -111,6 +112,7 @@ internal static void RemoveHistory(int TabID, int MaxVersion)
                 List<int> pages = GetAllByTabID(TabID).OrderByDescending(a => a.Version).Select(a => a.Version).Distinct().Take(MaxVersion).ToList();
                 if (pages.Count > 0)
                 {
+                    RemoveCustomPermissions(TabID, pages);
```

Comment:
Let's not use the word "Custom" because it creates confusion with Custom Blocks/Codes. Let's call it Entity if it's generic otherwise Section